src/main/java/core/basesyntax/strategy/impl/ReturnHandlerImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnHandlerImpl implements OperationHandler {
+
+    @Override
+    public void applyChanges(Transaction transaction) {
+        if (Storage.fruits.get(transaction.getFruit()) == null) {
```

Comment:
DAO
And why you cant return fruit if it isn't in storage? In this case you can add it as new fruit